Mathlib/Tactic/HelpCmd.lean

```diff
@@ -41,7 +41,9 @@ it will appear as a `(currently: true)` note next to the option.
 
 The form `#help option id` will show only options that begin with `id`.
 -/
-elab "#help " &"option" id:(ppSpace Parser.rawIdent)? : command => do
+syntax withPosition("#help " &"option" (colGt ppSpace Parser.rawIdent)?) : command
```

Comment:
Should there be a `colGt` around `option` too, to match the one around `attr` below?